### PR TITLE
Changes to threshold_cor

### PR DESCRIPTION
**Pull request by Larsvanderlaan**

### CODE REVIEW

File: cor_threshold/report_test.Rmd [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (benkeser) [inline cor_threshold/report_test.Rmd:1]: Please move this to `.gitignore` unless needed in repo

File: cor_threshold/figs/pointwise_CI/.Rapp.history [hunk 2]
```diff
@@ -0,0 +1,2 @@
+load("/Users/larsvanderlaan/correlates_reporting_forks/correlates_reporting-1/cor_threshold/figs/pointwise_CI/TABLE__belowDay29bindRBD_pointwiseCI.rds")
```
Comment (benkeser) [inline cor_threshold/figs/pointwise_CI/.Rapp.history:1]: `.gitignore`

**Comment by Larsvanderlaan** [comment]
A few more changes are coming later today

**Comment by benkeser** [comment]
> A few more changes are coming later today

Let me know when ready. Thanks.

**Comment by Larsvanderlaan** [comment]
Hi David,
This is ready to merge. There are no more changes.

**Pull request merged**